conf/messages

```diff
@@ -61,9 +61,10 @@ multiple.members.subheading = Multiple members
 ###### ERRORS ######################################################################################
 
 error.mandatory = Enter a {0}
-error.nino.invalid = Enter a valid National Insurance number
+error.nino.mandatory = Enter {0}’s {1}
```

Comment:
Two thoughts. Since the other pages are going to be changed, this error.nino.mandatory could be named something else since it is likely going to end up becoming the new 'error.mandatory' message. If it's not going to be used in other places, then it looks like the {1} parameter is always going to be the word 'nino' so that could be hard coded so it doesn't require two inputs.

Reply:
Made it generic with 2 arguments. updated message file and also validator and unit test.